Sources/SwiftCliCore/InputValidator.swift

```diff
@@ -15,6 +15,20 @@ public class InputValidator {
         return isItValid
     }
     
+
+    public func FindWidestRow(patternarray: [[String]]) -> Int {
+        var widest = -1;
+        
+        for row in (0...(patternarray.count - 1)){
+            if (patternarray[row].count > widest) {
+                widest = row
+            }
+        }
+        
+        return widest;
+    }
```

Comment:
Might be easier to follow the logic if you do:
```{swift}
rowLengths = patternarray.map { $0. count }
return rowLengths.firstIndex(of: rowLengths.max())
```

Reply:
Also, it doesn't look like anything is calling this function? Not sure if I'm just not seeing it.

Reply:
This is a function I was going to use to center charts correctly, but after our meeting on Friday and broke the task into 4 separate cards, I realized it probably wasn't going to be necessary. I'm going to just get rid of it for now.